lib/temporal/parent_close_policy.rb

```diff
@@ -0,0 +1,20 @@
+require 'temporal/errors'
+
+module Temporal
+  # See https://docs.temporal.io/docs/content/what-is-a-parent-close-policy/ for more information
+  class ParentClosePolicy
+    class InvalidParentClosePolicy < ClientError; end
+
+    attr_reader :policy
+
+    def initialize(policy)
+      @policy = policy
+    end
+
+    def validate!
+      unless %i[terminate abandon request_cancel].include?(@policy)
```

Comment:
Can you please define these as constants so they can be used without hardcoding symbols?